parity-util-mem/src/lib.rs

```diff
@@ -24,23 +24,37 @@ cfg_if::cfg_if! {
 		/// Global allocator
 		#[global_allocator]
 		pub static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
+
+		mod memory_stats_jemalloc;
+		use memory_stats_jemalloc as memory_stats;
```

Comment:
should it be pub use?

Reply:
I deliberately made it so that it's private.

The idea is to have a single facade which is always compiled and defines the public API, and an internal `memory_stats` module to which the facade calls into.

The alternative would be to just directly expose the `memory_stats` without a facade, but I personally don't like that very much since then there's nothing *within* the crate to enforce that the public API is always the same regardless of the selected backend.

Reply:
Ah, sorry, I missed the pub facade below somehow. Makes sense!